browser/src/control/Control.NavigatorPanel.ts

```diff
@@ -397,6 +401,30 @@ class NavigatorPanel extends SidebarBase {
 		this.builder.build(wrapper, [data], false);
 	}
 
+	createPlaceholderLabel() {
+		const quickFindPanel =
+			this.quickFindWrapper.querySelector('#QuickFindPanel');
+		this.placeholderLabel = L.DomUtil.create(
+			'label',
+			'jsdialog sidebar ui-text',
+			quickFindPanel,
+		);
+		this.placeholderLabel.id = 'quickfind-placeholder';
+		this.placeholderLabel.textContent = _(
+			'Type in the search box to find anything in your document',
+		);
+		this.placeholderLabel.setAttribute('aria-hidden', 'true');
+	}
+
+	placeholderLabelVisibility(searchTerm: string) {
+		if (!this.placeholderLabel) return;
```

Comment:
could be just JSDialog event:
`[{action: 'action', ](jsdialog: { "jsontype": "navigator", "action": "action", "id": 0, "data": { "control_id": "<HERE WIDGET ID>", "action_type": "hide"}})`

^ something like that